packages/assets-controllers/src/TokensController.ts

```diff
@@ -656,136 +607,32 @@ export class TokensController extends BaseController<
     asset: Token,
     type: string,
     interactingAddress?: string,
-  ): Promise<AssetSuggestionResult> {
+  ): Promise<void> {
+    if (type !== 'ERC20') {
```

Comment:
At a minimum, is this worth putting in an internal constant? Or should we go all out and create an exported enum to use for the `type` argument in order to discourage these errors?

Reply:
There's a constant already defined for `"ERC20"` in `controller-utils`.

The enum for types sounds good, but we'd first have to look a bit deeper into what is this controller used for, because even though there are mentions to `ERC721`, there's a whole different controller for NFTs.